<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiMainWindowClass</class>
 <widget class="QMainWindow" name="UiMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Langner_Laskowski_Projekt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="fishcard_List_Label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#ffffff;&quot;&gt;Lista Fiszek&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="Fishka_list_Box">
      <property name="styleSheet">
       <string notr="true">QComboBox {
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid #000000;  /* Czarna obramówka */
    background-color: #808080;  /* Szare wypełnienie */
    color: black;  /* Biały kolor tekstu */
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fishka_Label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#ffffff;&quot;&gt;Fiszka&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="fishcard_Box">
      <property name="styleSheet">
       <string notr="true">QTextEdit {
	font: 22pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid  #000000;  /* Czarna obramówka */
    background-color: #808080;  /* Szare wypełnienie */
}</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoNone</set>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:22pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="flip_Fishka_Button">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid #000000;  /* Czarna obramówka */
    background-color: #808080;  /* Szare wypełnienie */
    color: black;  /* Biały kolor tekstu */
}</string>
      </property>
      <property name="text">
       <string> Obróć fiszkę</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="edit_fishka_Button">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid #000000;  /* Czarna obramówka */
    background-color: #808080;  /* Szare wypełnienie */
    color: black;  /* Biały kolor tekstu */
}</string>
      </property>
      <property name="text">
       <string>Edytuj fiszkę</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_Fishka_Button">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid #000000;  /* Czarna obramówka */
    background-color: #808080;  /* Szare wypełnienie */
    color: black;  /* Biały kolor tekstu */
}</string>
      </property>
      <property name="text">
       <string>Dodaj fiszkę</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="subb_Fishka_Button">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	font: 700 12pt &quot;Microsoft YaHei UI&quot;;
    border: 3px solid #000000;  /* Czarna obramówka */
    background-color: #808080;  /* Szare wypełnienie */
    color: black;  /* Biały kolor tekstu */
}</string>
      </property>
      <property name="text">
       <string>Usuń fiszkę</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>add_Fishka_Button</sender>
   <signal>clicked()</signal>
   <receiver>UiMainWindowClass</receiver>
   <slot>addButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subb_Fishka_Button</sender>
   <signal>clicked()</signal>
   <receiver>UiMainWindowClass</receiver>
   <slot>subbButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>flip_Fishka_Button</sender>
   <signal>clicked()</signal>
   <receiver>UiMainWindowClass</receiver>
   <slot>flipButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Fishka_list_Box</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>UiMainWindowClass</receiver>
   <slot>displaySelectedFishka(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_fishka_Button</sender>
   <signal>clicked()</signal>
   <receiver>UiMainWindowClass</receiver>
   <slot>fishkaEdit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addButton_clicked()</slot>
  <slot>subbButton_clicked()</slot>
  <slot>flipButton_clicked()</slot>
  <slot>displaySelectedFishka(int)</slot>
  <slot>fishkaEdit_clicked()</slot>
 </slots>
</ui>
